src/inject.js

```diff
@@ -269,7 +275,16 @@ export const injectAndGetClassName = (
     if (processedStyleDefinitions.classNameBits.length === 0) {
         return "";
     }
-    const className = processedStyleDefinitions.classNameBits.join("-o_O-");
+
+    let className;
+    if (process.env.NODE_ENV === 'production') {
+        className = processedStyleDefinitions.classNameBits.length === 1 ?
+            `_${processedStyleDefinitions.classNameBits[0]}` :
```

Comment:
I'm not sure what the advantage of that would be. Also, before this PR the hashes were different lengths--for example, from our homepage:

```
image_ay4wjb-o_O-background_1h6n1zu-o_O-fadeIn_3jddj2-o_O-backgroundSize_cover_176vplr
```
--so I imagine they still won't be the same size.